File: src/main/java/sc02/Q06.java
package sc02;

/*
Input
ksekkset

Output
kset
 */

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Q06 {
    public static void main(String[] args) {
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String line = br.readLine();
            StringBuilder sb = new StringBuilder();
            for (int i = 0; i < line.length(); i++) {
                if (line.indexOf(line.charAt(i)) == i) {
                    sb.append(line.charAt(i));
                }
            }
            System.out.println(sb);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: src/main/kotlin/sc02/Q06.kt
package jp.co.kopher.sc02

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

fun main() {
    try {
        BufferedReader(InputStreamReader(System.`in`)).use { br ->
            val line = br.readLine()
            val sb = StringBuilder()
            for (i in line.indices) {
                if (line.indexOf(line[i]) == i) {
                    sb.append(line[i])
                }
            }
            println(sb.toString())
        }
    } catch (e: IOException) {
        e.printStackTrace()
    }
}